### PR TITLE
OSPP: Implementation of Domain-Specific Large Model Benchmarking Based on KubeEdge-Ianvs

### PR DESCRIPTION
**Pull request by IcyFeather233**
This PR is the implementation of https://github.com/kubeedge/ianvs/issues/95

### CODE REVIEW

**Review by hsj576 — CHANGES_REQUESTED** [review]
The "print" in the code should be replaced by "LOGGER.info" (from core.common.log import LOGGER ").

**Comment by IcyFeather233** [comment]
> The "print" in the code should be replaced by "LOGGER.info" (from core.common.log import LOGGER ").

@hsj576 Thank you for your advice! Now they are all replaced.

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Rebases are needed. 

1. There is a file with conflicts like docs/proposals/scenarios/llm-benchmarks/llm-benchmarks.md that needs to take care of. e.g., using rebase https://liwt31.github.io/2019/12/25/merge_rebase/.  

2. all commits need sign-off, which can also be solved by using rebase 

> Commit sha: [9b7f9c5](https://github.com/kubeedge/ianvs/pull/144/commits/9b7f9c51495b06557ae5f8f1d10f57f55df6dc22), Author: IcyFeather, Committer: IcyFeather; The sign-off is missing.
> Commit sha: [ce9f470](https://github.com/kubeedge/ianvs/pull/144/commits/ce9f470f0f4614fbdf06d90f857a6457ae9cddd3), Author: IcyFeather, Committer: IcyFeather; The sign-off is missing.


> To add your Signed-off-by line to every commit in this branch:
> 
> Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
> In your local branch, run: git rebase HEAD~11 --signoff
> Force push your changes to overwrite the branch: git push --force-with-lease origin dev

**Review by MooreZheng — CHANGES_REQUESTED** [review]
> Rebases are needed.
> 
> 1. There is a file with conflicts like docs/proposals/scenarios/llm-benchmarks/llm-benchmarks.md that needs to take care of. e.g., using rebase https://liwt31.github.io/2019/12/25/merge_rebase/.
> 2. all commits need sign-off, which can also be solved by using rebase
> 
> > Commit sha: [9b7f9c5](https://github.com/kubeedge/ianvs/pull/144/commits/9b7f9c51495b06557ae5f8f1d10f57f55df6dc22), Author: IcyFeather, Committer: IcyFeather; The sign-off is missing.
> > Commit sha: [ce9f470](https://github.com/kubeedge/ianvs/pull/144/commits/ce9f470f0f4614fbdf06d90f857a6457ae9cddd3), Author: IcyFeather, Committer: IcyFeather; The sign-off is missing.
> 
> > To add your Signed-off-by line to every commit in this branch:
> > Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
> > In your local branch, run: git rebase HEAD~11 --signoff
> > Force push your changes to overwrite the branch: git push --force-with-lease origin dev

Reminder: There are three CI issues that remain to be resolved, mainly for Pylint, see https://github.com/kubeedge/ianvs/actions/runs/11268242806

**Comment by IcyFeather233** [comment]
@MooreZheng The similar pylint issues are described in https://github.com/kubeedge/ianvs/issues/152

To solve this, sedna and ianvs need to corporate. We discussed this problem in the last week zoom meeting, and it seems that it should be discussed more to find a final solution.

**Comment by MooreZheng** [comment]
@hsj576 might also want to review this project

File: examples/government/singletask_learning_bench/objective/testenv/acc.py
Comment (MooreZheng) [inline examples/government/singletask_learning_bench/objective/testenv/acc.py:20]: In the acc.py, need to replace Chinese characters with English.

File: examples/government/singletask_learning_bench/subjective/testenv/llm_judgement.py
Comment (MooreZheng) [inline examples/government/singletask_learning_bench/subjective/testenv/llm_judgement.py:22]: In the llm_judgement.py, we need to replace Chinese characters with English.

File: examples/llm_simple_qa/README.md
Comment (MooreZheng) [inline examples/llm_simple_qa/README.md:17]: As the following examples, we also need to replace questions in Chinese with those in English.

File: examples/llm_simple_qa/testenv/acc.py
Comment (MooreZheng) [inline examples/llm_simple_qa/testenv/acc.py:20]: In the acc.py in examples, need to replace Chinese comments into English.

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Looks fine, but we need to refine the Chinese words as in English.

**Comment by IcyFeather233** [comment]
@MooreZheng I have changed Chinese words to English

File: examples/government/singletask_learning_bench/subjective/testenv/llm_judgement.py
Comment (MooreZheng) [inline examples/government/singletask_learning_bench/subjective/testenv/llm_judgement.py:23]: In the re.search of llm_judgement.py, we need to replace Chinese characters with English.

**Comment by MooreZheng** [comment]
Overall it looks fine. Any comment from @hsj576 before merging?

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Good. There is only one Chinese statement right now and we are very close to the final version.

Currently, this pull request contains 25 commits, which can make maintenance difficult, considering the number of contributors, pull requests, and their commits in KubeEdge Ianvs. In the final stage, @IcyFeather233 can squash the commits into one using [rebase techniques](https://blog.kinboy.wang/2019/04/09/Git-%E5%8E%8B%E7%BC%A9%E5%A4%9A%E4%B8%AAcommit%E4%B8%BA%E5%8D%95%E4%B8%AAcommit/).

**Comment by IcyFeather233** [comment]
@MooreZheng All commits merged into one, and all Chinese words changed to English

**Review by MooreZheng — CHANGES_REQUESTED** [review]
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/kubeedge/ianvs/pull/144/conflicts) or the  to resolve conflicts.
Conflicting files
core/testenvmanager/dataset/dataset.py

**Comment by IcyFeather233** [comment]
@MooreZheng conflicts are resolved

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Conflicts still exists 

This branch has conflicts that must be resolved
Use the [web editor](https://github.com/kubeedge/ianvs/pull/144/conflicts) or the  to resolve conflicts.
Conflicting files
core/testenvmanager/dataset/dataset.py

**Comment by MooreZheng** [comment]
Now conflicts have been solved

**Review by MooreZheng — APPROVED** [review]
All concerns are fixed. Well done!

**Comment by MooreZheng** [comment]
Related issue and PR:  
Phase 1 Gov LLM https://github.com/kubeedge/ianvs/issues/95 https://github.com/kubeedge/ianvs/pull/113 https://github.com/kubeedge/ianvs/pull/144
Phase 2 Gov RAG https://github.com/kubeedge/ianvs/issues/177 https://github.com/kubeedge/ianvs/pull/186 https://github.com/kubeedge/ianvs/pull/196

**Pull request merged**
